### PR TITLE
Update snippet links

### PR DESCRIPTION
**Pull request by tsmarvin**
Updated snippet links - The snippets still arent rendering for me but at least the links go to the right place.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/csharp/programming-guide/main-and-command-args/main-return-values.md [hunk 1]
```diff
@@ -11,11 +11,11 @@ ms.assetid: c2f5a1d8-1676-4bea-bc7e-44a97e72d5bc
 
 The `Main` method can return `void`:
 
- [!code-csharp[csProgGuideMain#12](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class3.cs#12)]
+ [!code-csharp[csProgGuideMain#12](~../../../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class3.cs#12)]
```
Comment (BillWagner) [inline docs/csharp/programming-guide/main-and-command-args/main-return-values.md:14]: Can you remove the `~` character and the first two `..` paths: here:

```suggestion
 [!code-csharp[csProgGuideMain#12](../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class3.cs#12)]
```

File: docs/csharp/programming-guide/main-and-command-args/main-return-values.md [hunk 2]
```diff
@@ -11,11 +11,11 @@ ms.assetid: c2f5a1d8-1676-4bea-bc7e-44a97e72d5bc
...
 
 It can also return an `int`:
 
- [!code-csharp[csProgGuideMain#13](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class3.cs#13)]
+ [!code-csharp[csProgGuideMain#13](~../../../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class3.cs#13)]
```
Comment (BillWagner) [inline docs/csharp/programming-guide/main-and-command-args/main-return-values.md:18]: same:

```suggestion
 [!code-csharp[csProgGuideMain#13](../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class3.cs#13)]
```

File: docs/csharp/programming-guide/main-and-command-args/main-return-values.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ This example uses [.NET Core](../../../core/index.yml) command-line tools. If yo
 
 Modify the `Main` method in *program.cs* as follows:
 
- [!code-csharp[csProgGuideMain#14](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class3.cs#14)]
+ [!code-csharp[csProgGuideMain#14](~../../../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class3.cs#14)]
```
Comment (BillWagner) [inline docs/csharp/programming-guide/main-and-command-args/main-return-values.md:28]: same:

```suggestion
 [!code-csharp[csProgGuideMain#14](../../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideMain/CS/Class3.cs#14)]
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks for making these fixes @tsmarvin 

I checked the review site, and I have a few suggestions that should fix the snippet rendering. Once you accept those suggestions, I'll :shipit:

Thanks again!

**Comment by Youssef1313** [comment]
@BillWagner, I see the current live already renders the snippets correctly.

**Comment by tsmarvin** [comment]
Apologies for the delay - work has been hectic.

**Comment by adegeo** [comment]
@BillWagner I think these got ninja fixed by something else. As @Youssef1313 said, these are currently displaying correctly.

**Comment by BillWagner** [comment]
I'll close this because all snippets are rendering correctly.

**Pull request closed without merging**